cmake/ProjectWasm3.cmake

```diff
@@ -16,17 +16,18 @@ set(wasm3_library ${binary_dir}/source/${CMAKE_STATIC_LIBRARY_PREFIX}m3${CMAKE_S
 ExternalProject_Add(wasm3
     EXCLUDE_FROM_ALL 1
     PREFIX ${prefix}
-    DOWNLOAD_NAME wasm3-ext-api-v2.tar.gz
+    DOWNLOAD_NAME wasm3-ext-api-v4.tar.gz
     DOWNLOAD_DIR ${prefix}/downloads
     SOURCE_DIR ${source_dir}
     BINARY_DIR ${binary_dir}
-    URL https://github.com/axic/wasm3/archive/ext-api-v2.tar.gz
-    URL_HASH SHA256=6163efd2f9c5088a6c0a7e5c8f815871d8f70af5de3240c52be6c9473411fbdf
+    URL https://github.com/axic/wasm3/archive/ext-api-v4.tar.gz
+    URL_HASH SHA256=5ceb543ef5f44d563cef639701d35616c323663a3b7e41996b4bd56ab59c941c
     CMAKE_ARGS
     -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
     -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
     -DCMAKE_BUILD_TYPE=Release
-    -DBUILD_WASI_SUPPORT=FALSE
+    -DBUILD_WASI=none
+    -DBUILD_PORTABLE=ON
```

Comment:
Why do you insist on disabling `-march=native`?

Reply:
It was a necessity because otherwise bench was crashing (binary was built on a different VM than run). I asked you a while ago whether that is still the case but you didn't respond.

However, I think it would be fair to either build all engines with `-march=native` or none, to be on a level playing field. Otherwise we are chasing invalid targets.

Reply:
It is not an issue any more.

We can build with native for benchmarks, but you are overestimating the impact of it. Trying to modify other project's compiler flags seems to be dead end.

Reply:
That change is not an issue, the `m3_CallProper` is.

Reply:
To clarify, my main personal worry was to release fizzy 0.1.0 with my wasm3 fork. I want to delete my wasm3 fork, but if we release and I delete that, that means fizzy 0.1.0 is "broken".

Couldn't we alternatively just keep the changes as a patch in `ProjectWasm3.cmake`?